### ISSUE TITLE
Let's drop the recommendation of scope

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-tutorials#312]
These days Scope doesn't appear to add any value to new browsers and screen readers. And given the NVDA is free I don't think there is a good use case to get authors to spend the extra 15 minutes a table to add scope for old screen readers. 

http://www.w3.org/WAI/tutorials/tables/two-headers/

It suggest we drop it and just tell them to use proper th tags. My testing is here.
http://davidmacd.com/blog/csun/files/test-scope-tables.html

**Comment by Ryladog** [comment]
+1

 

 

 

* katie *

 

Katie Haritos-Shea 
Senior Accessibility SME (WCAG/Section 508/ADA/AODA)

 

Cell: 703-371-5545 |  <mailto:ryladog@gmail.com> ryladog@gmail.com | Oakton, VA |  <http://www.linkedin.com/in/katieharitosshea/> LinkedIn Profile | Office: 703-371-5545

**Comment by yatil** [comment]
@DavidMacDonald, thanks for your feedback, I think the complicated table in this example needs scope="colgroup" for the header row…

**Comment by WobleDuck** [comment]
I am -1.  First there is the upper left corner ambiguity that is not
deterministic. It fails 1.3.1. Screen readers guess the scope wrong a lot.
Next, it is good practice. Just because screen readers don't use useful
data doesn't mean it is not useful.

Wayne

**Comment by sharronrush** [comment]
Confirm with @DavidMacDonald @yatil and @WobleDuck to close this 2 1/2 year old issue?
